src/main/java/tornadofx/Builders.kt

```diff
@@ -109,6 +109,12 @@ fun Pane.progressbar(property: Property<Double>, op: (ProgressBar.() -> Unit)? =
     progressProperty().bind(property)
 }
 
+fun Pane.slider(min: Double? = null, max: Double? = null, value: Double? = null, op: (Slider.() -> Unit)? = null) = opcr(this, Slider().apply {
```

Comment:
In that case, I agree. regarding the `orientation` parameter I'm not sure I feel one way or the other. That can always be done in the `op` block but we try to streamline common usages which I guess this could be. What do you think @edvin?